<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainApp</class>
 <widget class="QMainWindow" name="MainApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>624</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainApp</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>624</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuA">
     <property name="title">
      <string>A</string>
     </property>
     <widget class="QMenu" name="menuB">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="title">
       <string>B</string>
      </property>
      <addaction name="actionC"/>
      <addaction name="actionD"/>
     </widget>
     <widget class="QMenu" name="menuE">
      <property name="title">
       <string>E</string>
      </property>
      <addaction name="actionF"/>
     </widget>
     <addaction name="menuB"/>
     <addaction name="menuE"/>
    </widget>
    <addaction name="menuA"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionC">
   <property name="text">
    <string>C</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionD">
   <property name="text">
    <string>D</string>
   </property>
  </action>
  <action name="actionF">
   <property name="text">
    <string>F</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
